releasenotes/notes/rabbit-disable-ha-9b150bcc9e4d4f54.yaml

```
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    RabbitMQ will now use quorum queues by default, rather than traditional HA.
```

"traditional HA" isn't really the term we've used in the past, and I don't think there's any merit in saying it tbh.
```suggestion
    RabbitMQ will now use quorum queues by default.
```

---

Removed, see https://github.com/stackhpc/stackhpc-kayobe-config/pull/1224/files